internal/resource/configmap.go

```diff
@@ -90,116 +91,115 @@ func (builder *ServerConfigMapBuilder) Update(object client.Object) error {
 		return err
 	}
 
+	cfgUserConfiguration := ini.Empty(ini.LoadOptions{})
+	userConfigurationSection := cfgUserConfiguration.Section("")
+
 	if builder.Instance.TLSEnabled() {
-		if err := cfg.Append([]byte(defaultTLSConf)); err != nil {
```

Comment:
Naming suggestion:
1)  `cfgUserConfiguration ` to `userConfiguration `
2) `cfg` to `operatorConfiguration`?